src/main/java/frc/robot/commands/CollectNote.java

```diff
@@ -97,24 +115,24 @@ public void execute() {
     
     SmartDashboard.putNumber("CollectNote/calculated radians per second", txController.calculate(tx));
     SmartDashboard.putNumber("CollectNote/calculated forward meters per second", tyController.calculate(ty));
+    SmartDashboard.putBoolean("CollectNote/closeNote", closeNote);
+    SmartDashboard.putNumber("CollectNote/runsInvalid", runsInvalid);
     // drives the robot forward faster if the object is higher up on the screen, and turns it more based on how far away the object is from x=0
-    drivetrain.drive(new ChassisSpeeds(
-      tyLimiter.calculate(-20/Math.abs(tx)),
-      0,
-      txController.calculate(tx)));
   }
   
 
 
   // Called once the command ends or is interrupted.
   @Override
   public void end(boolean interrupted) {
+    drivetrain.pointWheelsInward();
```

Comment:
Why? Does this not then require extra rotation from your drivetrain to the next path?

Reply:
The command that often runs after this is the autoShoot note. The autoShoot note doesn't use the drivetrain, so if the robot didn't get stopped here, it would coast to a stop, which is sometimes a few feet of movement.